rio_tiler_pds/modis/aws/modis_astraea.py

```diff
@@ -0,0 +1,137 @@
+"""AWS MODIS reader."""
+
+from typing import Dict, Type
+
+import attr
+
+from rio_tiler.errors import InvalidBandName
+from rio_tiler.io import BaseReader, COGReader
+
+from ...reader import MultiBandReader
+from ..modland_grid import tile_bbox
+from ..utils import sceneid_parser
+
+MCD43A4_BANDS = (
+    "B01",
+    "B01qa",
+    "B02",
+    "B02qa",
+    "B03",
+    "B03qa",
+    "B04",
+    "B04qa",
+    "B05",
+    "B05qa",
+    "B06",
+    "B06qa",
+    "B07",
+    "B07qa",
+)
+
+MODIS_BANDS = (
+    "B01",
+    "B02",
+    "B03",
+    "B04",
+    "B05",
+    "B06",
+    "B07",
+    "B08",
+    "B09",
+    "B10",
+    "B11",
+    "B12",
+)
+
+modis_valid_bands = {
+    "MCD43A4": MCD43A4_BANDS,
+    "MOD11A1": MODIS_BANDS,
+    "MYD11A1": MODIS_BANDS,
+    "MOD13A1": MODIS_BANDS,
+    "MYD13A1": MODIS_BANDS,
+}
+
+MOD11A1_MYD11A1_PREFIX = {
+    "B01": "LSTD_",
+    "B02": "QCD_",
+    "B03": "DVT_",
+    "B04": "DVA_",
+    "B05": "LSTN_",
+    "B06": "QCN_",
+    "B07": "NVT_",
+    "B08": "NVA_",
+    "B09": "E31_",
+    "B10": "E32_",
+    "B11": "CDC_",
+    "B12": "CNC_",
+}
+
+MOD13A1_MYD13A1_PREFIX = {
+    "B01": "NDVI_",
+    "B02": "EVI_",
+    "B03": "VIQ_",
+    "B04": "RR_",
+    "B05": "NIRR_",
+    "B06": "BR_",
+    "B07": "MIRR_",
+    "B08": "VZA_",
+    "B09": "SZA_",
+    "B10": "RAA_",
+    "B11": "CDOY_",
+    "B12": "PR_",
+}
```

Comment:
I created those list of bands/prefix by just checking some data on AWS ... so I hope they are constant!